db/schema.rb

```diff
@@ -35,4 +35,38 @@
     t.boolean  "is_featured"
   end
 
+  create_table "installs", force: :cascade do |t|
```

Comment:
Same for this content of the db/schema.rb file, which is auto updated when you run migrations...